### PR TITLE
LPS-124157 Avoid initializing another tooltip

### PR DESCRIPTION
**Pull request by julien**
This fixes the "duplicated" tooltips in
"Content & Data" > "Forms"

**Test plan**
- Deploy the `dynamic-data-mapping-form-builder` module
- Assert that tooltips are working as expected

**Before**

**After**

**Additionally**:

- Some elements don't have a `title` attribute (like most "ellipsis" buttons) and will need to be updated to show the correct text in tooltips.

- I discovered this other [Tooltip component](https://github.com/liferay/liferay-portal/blob/dd77a9a725e3403b98c95e1126b2db1188f575f3/modules%2Fapps%2Fdynamic-data-mapping%2Fdynamic-data-mapping-form-field-type%2Fsrc%2Fmain%2Fresources%2FMETA-INF%2Fresources%2Fcomponents%2FTooltip%2FTooltip.es.js) and I'm not sure it's needed.

- While working on the issue, I noticed that [the following component](https://github.com/liferay/liferay-portal/blob/93b71aae5e9964c814151c5fba4d9b959288be19/modules%2Fapps%2Fdynamic-data-mapping%2Fdynamic-data-mapping-form-renderer%2Fsrc%2Fmain%2Fresources%2FMETA-INF%2Fresources%2Fjs%2Fcomponents%2FField%2FReactFieldAdapter.es.js#L33-L34) generates a React warning which looks like this:

According to @diegonvs this will be removed soon, but I thought it would be good mentioning

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Review by wincent — APPROVED** [review]
Seems legit?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ import 'clay-button';
 import 'clay-label';
 
 import 'clay-dropdown';
-import ClayTooltip from 'clay-tooltip';
```
Comment (jbalsas) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js:20]: I was going to say this should also be removed from `dependencies` in the `package.json` of the module, but a quick check reveals it isn't even declared 🙈

Comment (julien) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js:20]: yeah I thought I'd find it in there too

**Comment by jbalsas** [comment]
LGTM too... if green, let's just push this forward since the issue got pinned on us and they're on cc anyways.

**Comment by julien** [comment]
@jbalsas I'll remove 
> Important: This PR is meant to be sent to @liferay-forms once it passes our internal review
From the commit description just to avoid confusion

**Comment by kresimir-coko** [comment]
Not terrible PR 👍

**Comment by kresimir-coko** [comment]
ci:forward

**Pull request closed without merging**